################################################################################
Concuerror started with options:
  [{after_timeout,infinity},
   {assume_racing,false},
   {delay_bound,infinity},
   {depth_bound,5000},
   {entry_point,{messages_2,test,[]}},
   {files,["/home/stavros/git/Concuerror/tests/suites/dpor_tests/src/messages_2.erl"]},
   {ignore_error,[]},
   {ignore_first_crash,true},
   {instant_delivery,false},
   {non_racing_system,[]},
   {optimal,true},
   {print_depth,20},
   {quiet,true},
   {report_unknown,true},
   {scheduling,round_robin},
   {strict_scheduling,false},
   {symbolic_names,true},
   {timeout,infinity},
   {treat_as_normal,[]}]
################################################################################
Erroneous interleaving 1:
* At step 14 process P.1 exited abnormally
    Reason:
      {{badmatch,a},
       [{messages_2,'-test/0-fun-0-',0,
                    [{file,"/home/stavros/git/Concuerror/tests/suites/dpor_tests/src/messages_2.erl"},
                     {line,23}]}]}
    Stacktrace:
      [{messages_2,'-test/0-fun-0-',0,
                   [{file,"/home/stavros/git/Concuerror/tests/suites/dpor_tests/src/messages_2.erl"},
                    {line,23}]}]
--------------------------------------------------------------------------------
Interleaving info:
   1: P: P.1 = erlang:spawn(erlang, apply, [#Fun<messages_2.0.54529166>,[]])
    in erlang.erl line 2329
   2: P: P.2 = erlang:spawn(erlang, apply, [#Fun<messages_2.1.54529166>,[]])
    in erlang.erl line 2329
   3: P: a = P.1 ! a
    in messages_2.erl line 30
   4: P: special = P.1 ! special
    in messages_2.erl line 31
   5: P: exits normally
   6: P.2: c = P.1 ! c
    in messages_2.erl line 28
   7: P.2: exits normally
   8: P.1: receive timeout expired after 0 ms
    in messages_2.erl line 13
   9: Message (c) from P.2 reaches P.1
  10: Message (a) from P reaches P.1
  11: Message (special) from P reaches P.1
  12: P.1: receives message (special)
    in messages_2.erl line 16
  13: P.1: receives message (c)
    in messages_2.erl line 23
  14: P.1: exits abnormally ({{badmatch,a},[{messages_2,'-test/0-fun-0-',0,[{file,[47,104,111,109,101,47,115,116,97|...]},{line,23}]}]})
################################################################################
Concuerror Info:
--------------------------------------------------------------------------------
Instrumented: messages_2
Instrumented: io_lib
Instrumented: erlang
################################################################################
Done! (Exit status: completed)
  Summary: 1 errors, 4/4 interleavings explored
